integration_tests/telemetry_async.cpp

```diff
@@ -40,6 +41,7 @@ static bool _received_ground_speed = false;
 static bool _received_gps_info = false;
 static bool _received_battery = false;
 static bool _received_rc_status = false;
+static bool _received_status_text = false;
```

Comment:
I don't know if we should check for this because it's not really guaranteed that we receive any.